### PR TITLE
Development

### PR DESCRIPTION
**Pull request by AllBill127**

### CODE REVIEW

File: pages/extraPage1.html [hunk 1]
```diff
@@ -12,8 +12,13 @@
         <h2>Welcome to the FAQ page! I'm still working on it!</h2>
         
         <details>
-            <summary>Question 1</summary>
-            <p>Answer 1</p>
+            <summary>What is this website?</summary>
+            <p>
+                This Website is the collaborative result of an assignment
+                worked on by the team of Aleksandras Bilevičius, 
+                Aleksandras Bilevičius, Ashley Nyasha Mlambo, and
```
Comment (AllBill127) [inline pages/extraPage1.html:19]: duplicate name. missing Michael

File: styles/faq.css [hunk 2]
```diff
@@ -24,7 +24,7 @@ details > summary {
 details > p {
 	border-radius: 0 10px 10px 10px;
 	background-color: #ddd;
```
Comment (AllBill127) [inline styles/faq.css:26]: should use colors from other pages for consistency

File: styles/index.css [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+    Note: name must have '--' prefix */
+:root {
+    --review-count: 6;
+}
+
+body {
+	font-family: 'Roboto', sans-serif;
+	margin: 0;
+}
+
```
Comment (ying-lenny) [inline styles/index.css:13]: Duplicate code, unnessecary and can be deleted

File: styles/index.css [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+	background-color: #3882F6;
+	color: whitesmoke;
+	font-family: 'Lato', sans-serif;
+	font-size: .8em;
+}
+
+.headerImage{
+	height: 190px;
+	width: 300px;
+	padding-right: 10%;
```
Comment (ying-lenny) [inline styles/index.css:129]: Can be removed

Comment (ying-lenny) [inline styles/index.css:129]: Can be removed

**Pull request merged**